content/docs/details/create-proxy/structure/header/index.md

```diff
@@ -0,0 +1,21 @@
+---
+title: Create a Proxy
+publishedDate: '2022-07-18T14:00:00.0Z'
+description: Creating a proxy.
+
+category: header
+---
+
+Accessing your own self-hosted solutions and 3rd party product APIs securely from Roadie is crucial in ensuring security of data and systems. Roadie provides two kinds of proxy services to handle these scenarios. 
+
+With the Proxy feature, there’s an easier way. In this tutorial, we’re going to:
```

Comment:
Easier than what?

Reply:
I would just remove the line starting with `With the Proxy`

Reply:
This is from the template for TI tutorials. TI makes it easier re: "This can be difficult to track the information stored as a decorator or fragment which you would like to be tracked. "